### PR TITLE
First version of Trace app UI

### PR DESCRIPTION
**Pull request by sshivanii**
*Issue #, if available:*

*Description of changes:*
This PR adds the first version of the Trace App UI.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kowshikn — APPROVED** [review]

File: examples/odfe-pipes-trace-app/client/client.py [hunk 1]
```diff
@@ -96,65 +162,100 @@ def cleanupDB():
...
+                data=[
+                    ("banana", 2),
+                    ("orange", 3),
+                    ("apple", 1),
+                ],
+            )
+            assert checkoutAPIRequest.status_code == 200
+            return get_ref_link("Checkout", "success", trace_id)
+    except:
+        return get_ref_link("Checkout", "failed", trace_id)
```
Comment (chenqi0805) [inline examples/odfe-pipes-trace-app/client/client.py:182]: I suggest the following if it works as expected:

```
with tracer.start_as_current_span("client_checkout") as checkout_trace_group:
    trace_id = get_hexadecimal_trace_id(checkout_trace_group.get_context().trace_id)
    try:
        checkoutAPIRequest = post(
            "http://{}:8084/checkout".format(PAYMENT),
            data=[
                ("banana", 2),
                ("orange", 3),
                ("apple", 1),
            ],
        )
        assert checkoutAPIRequest.status_code == 200
        return get_ref_link("Checkout", "success", trace_id)
    except:
        return get_ref_link("Checkout", "failed", trace_id)
```

same below.

**Review by chenqi0805 — APPROVED** [review]
Overall LGTM. Just one codestyle nit. Nothing blocking

File: examples/odfe-pipes-trace-app/client/client.py (hunk 1 above)
Comment (kowshikn) [reply examples/odfe-pipes-trace-app/client/client.py:182]: Actually this would work, I will probably fix it the next time when we address this.

**Comment by sshivanii** [comment]
Closes #164

**Pull request merged**